### ISSUE TITLE
"firejail --list" does not show hardlinked firefox

### ISSUE THREAD

**Issue by shmu26** [netblue30/firetools#64]
I made a hard link to start firefox in firejail:
`sudo ln -s /usr/bin/firejail /usr/local/bin/firefox`
when I run `firejail --list` I don't get any results
but when I run `which firefox` I get `/usr/local/bin/firefox`

Manjaro, firejail 0.9.70-1

**Comment by rusty-snake** [comment]
Did you crate a hardlink or a softlink? (Your command creates a softlink.)
Hardlinks do not work because both links are the same file.

How did you start firefox? Was there an already running firefox instance?

**Comment by shmu26** [comment]
So it is a softlink, then, if that is what my command creates...
I started firefox by clicking on its icon in the applications menu.
There was no instance of firefox running.

**Comment by shmu26** [comment]
So it is a softlink, then, if that is what my command creates...
I started firefox by clicking on its icon in the applications menu.
There was no instance of firefox running.

Now I tried starting it by double-clicking in the icon in Firetools, and `firejail --list` shows it properly.

**Comment by shmu26** [comment]
I wonder if this has to do with wayland. I am running Gnome 42 with wayland.

**Comment by rusty-snake** [comment]
If manjaro uses the firefox packages from arch, you need to patch the .desktop file.

**Comment by shmu26** [comment]
Yes. patching  .desktop files, as per the Arch Wiki, was the answer. Thanks!
